### ISSUE TITLE
Add flag to have non-fancy aka non-coloured logs

### ISSUE THREAD

**Issue by lanzafame** [ipfs/ipfs-cluster#660]
#### Basic information

* [x] Version information (mark as appropiate):
  * [x] Master
  * [ ] Release candidate for next version
  * [ ] Latest stable version
  * [ ] An older version I should not be using
* [x] Type (mark as appropiate):
  * [ ] Bug
  * [ ] Feature request
  * [x] Enhancement

#### Description

Not all logging infrastructure supports coloured terminal text and it results in very difficult to parse log messages, i.e.:
```
[0;37m18:29:42.405 [31mERROR [0;34m  ipfshttp: [0merror posting to IPFS:<truncated>
```

It would be great if there was a config value to turn off the coloured logging, as this enhancement is aimed at cloud deployments over user-driven interactions with the cli tool, an env var or service.json toggle should be provided.

**Comment by hsanjuan** [comment]
Agreed, but not sure how easy. Maybe a cli flag would suffice, since logging is controlled from flags right (level wise)

**Comment by roignpar** [comment]
I looked a bit into this. It seems that https://github.com/ipfs/go-log can only be configured with env vars regarding color. To turn off log colors you must set `IPFS_LOGGING_FMT` to `nocolor`. e.g.:
```sh
IPFS_LOGGING_FMT=nocolor ipfs-cluster-service daemon
```
I tried it and it works.

Maybe it would be enough to add this detail to the documentation?

If you still want to add a cli flag or a config value, I would be happy to make the changes!
